packages/ui/src/components/select/select.tsx

```diff
@@ -0,0 +1,120 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import { cx } from "classix";
+import React, { ComponentProps } from "react";
+import {
+  Button as BaseButton,
+  FieldError as BaseFieldError,
+  Form as BaseForm,
+  Label as BaseLabel,
+  ListBox as BaseListBox,
+  ListBoxItem as BaseListBoxItem,
+  Popover as BasePopover,
+  Select as BaseSelect,
+  SelectValue as BaseSelectValue,
+} from "react-aria-components";
+
+export const Form = (properties: ComponentProps<typeof BaseForm> & { className?: string | undefined }) => {
+  const { className = "", ...rest } = properties;
+  const mergedClassName = cx("relative w-full", className);
+  return <BaseForm className={mergedClassName} {...rest} />;
+};
+
+export const Select = (
+  properties: ComponentProps<typeof BaseSelect> & { className?: string | undefined },
+) => {
+  const { className = "", ...rest } = properties;
+  const mergedClassName = cx("group w-full focus:outline-none", className);
+  return <BaseSelect className={mergedClassName} {...rest} />;
+};
+
+export const ListBox = (
```

Comment:
I'm thinking we should prefix the components with `Select`. e.g.: `SelectListBox`. It's probably more ergonomic because consumers will likely have to do lots of renaming otherwise.